test/prototype/mx_formats/test_mx_tensor.py

```diff
@@ -258,18 +264,21 @@ def test_to_mx_from_mx_compile_numerics(elem_dtype, hp_dtype, all_zeros):
 
     to_dtype_c = torch.compile(to_dtype, fullgraph=True)
 
+    use_fp4_custom_triton_dequant_kernel = False
     x_mx_dq = to_dtype(
         x_mx._data,
         x_mx._scale_e8m0,
         x_mx._elem_dtype,
         x_mx._block_size,
         hp_dtype,  # noqa: E501
+        use_fp4_custom_triton_dequant_kernel,
```

Comment:
Why is this needed? I did a quick pass through the RFC and didn't see any reference to this functionality

Reply:
we'll probably delete it in a future PR.  The current PR is a refactor with no logic changes.